components/camel-bindy/src/main/java/org/apache/camel/dataformat/bindy/csv/BindyCsvDataFormat.java

```diff
@@ -16,6 +16,19 @@
  */
 package org.apache.camel.dataformat.bindy.csv;
 
+import org.apache.camel.Exchange;
```

Comment:
mvn clean install -Psourcecheck, possibly the import order is not proper.

Reply:
I'll double-check this.  I had ran this goal before committing and I didn't notice any errors.  I'll look again tonight.

Reply:
I may have anticipated that this would fail the build.  I've seen the errors and have corrected them with the latest commits.